hls4ml/templates/oneapi/firmware/nnet_utils/nnet_sepconv2d_latency.h

```diff
@@ -0,0 +1,77 @@
+#ifndef NNET_SEPARABLE_CONV2D_LATENCY_H_
+#define NNET_SEPARABLE_CONV2D_LATENCY_H_
+
+#include "nnet_common.h"
+#include "nnet_conv2d_resource.h"
+#include "nnet_mult.h"
+
+namespace nnet {
+
+template <class data_T, class res_T, typename CONFIG_T>
+void depthwise_conv_2d_latency_cl(const data_T &data, res_T &res, const typename CONFIG_T::weight_t &weights,
```

Comment:
Currently oneAPI, like Quartus, is always resource. We can revisit, but as is, this would never be used.